src/top/core/models.py

```diff
@@ -84,6 +84,14 @@ class OidcConfiguration(BaseModel):
         description="List of JWS signing algorithms"
         " supported by the OP for the UserInfo endpoint",
     )
+    authorization_endpoint: AnyHttpUrl = Field(
+        default=AnyHttpUrl("http://localhost:8080/authorize"),
+        description="URL of the OP's Authorization Endpoint",
+    )
+    token_endpoint: AnyHttpUrl = Field(
+        default=AnyHttpUrl("http://localhost:8080/authorize"),
```

Comment:
This must be `token` right?
```suggestion
        default=AnyHttpUrl("http://localhost:8080/token"),
```